### ISSUE TITLE
Hmmm

### ISSUE THREAD

**Issue by riverDot666** [0xb11a1/yetAnotherObfuscator#3]
just convert string to Base64 , 👍

**Comment by 0xb11a1** [comment]
Yes because currently this is enough to bypass windows defender, if this changes in the future i will implement a different technique.

**Comment by 0xb11a1** [comment]
@psoutis can you please share more details

**Comment by Resisst** [comment]
> @psoutis can you please share more details

windows defender believe it or not can base64 decode strings lol, it sometimes even mentions the fact that its base64 decoded in some instances such as where there is a fully encoded malware payload and after decoding it defender determines its malware.

it won't be too much harder to just encrypt the strings, you dont even have to use a hardcoded key you can use different keys for each string.

im willing to work on it and create a pull request if you're down.

**Comment by Resisst** [comment]
made PR: https://github.com/0xb11a1/yetAnotherObfuscator/pull/5
